### DISCUSSION TITLE
linker-wrapper.sh` failed: exit status: 127

### DISCUSSION THREAD

**Discussion by ysjava** [shadowsocks/shadowsocks-android#2923] [General]
error: linking with `/Users/test/Desktop/Projects/shadowsocks-android/build/linker-wrapper/linker-wrapper.sh` failed: exit status: 127
linking with `/Users/test/Desktop/Projects/shadowsocks-android/build/linker-wrapper/linker-wrapper.sh` failed: exit status: 127

  |
  = note: "/Users/test/Desktop/Projects/shadowsocks-android/build/linker-wrapper/linker-wrapper.sh" ....
  = note: /Users/test/Desktop/Projects/shadowsocks-android/build/linker-wrapper/linker-wrapper.sh: line 4: python: command not found

`linker-wrapper.sh` :

`#!/usr/bin/env bash

# Invoke linker-wrapper.py with the correct Python command.
"${RUST_ANDROID_GRADLE_PYTHON_COMMAND}" "${RUST_ANDROID_GRADLE_LINKER_WRAPPER_PY}" "$@"`

好像是跟python有关。 我不了解它，请问怎么解决
It seems to have something to do with Python. I don't understand it. How can I solve it

**Comment by dev4u** [comment]
`python: command not found`
确认系统已经安装有python。

**Comment by ysjava** [comment]
> `python: command not found` 确认系统已经安装有python。

python3不行，python2可以，当然也可能是我环境配置错误

**Comment by dev4u** [comment]
python3需要在core的build.gradle.kts文件里面指定。

**Comment by ysjava** [comment]
> python3需要在core的build.gradle.kts文件里面指定。

请问这个具体是怎么写的？
